.gitignore

```diff
@@ -12,3 +12,4 @@ yarn-error.log
 .media
 .env
 .changelog
+.vscode/
```

Comment:
This belongs into your personal `.gitignore`. I can show you how to do this.

Reply:
I think I found [how you do it](https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b)?

Reply:
👍🏾